<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page4adv</class>
 <widget class="QDialog" name="Page4adv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Populate VDF Data: Advanced Options</string>
  </property>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>58</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression Level</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>102</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of     Threads</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="numThreadsSpinner">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>108</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Number of execution threads (0 =&gt; # processors).</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="refinementLevelSpinner">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refinement levels saved. 0=&gt;coarsest, 1=&gt;next refinement, etc. -1=&gt;finest.</string>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="compressionLevelSpinner">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>56</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Compression levels saved. 0 =&gt; coarsest, 1 =&gt; next refinement, etc. -1  =&gt; all levels defined by the .vdf file. </string>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>14</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Refinement Level</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreDefaultButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>146</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Default Values</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
